src/groovy/com/unifina/api/ListParams.groovy

```diff
@@ -15,6 +15,7 @@ abstract class ListParams {
 	Integer offset = 0
 	Boolean grantedAccess = true
 	Boolean publicAccess = false
+	Boolean includePermissions = false
```

Comment:
Is it possible or would it make sense to use primtive boolean type here to get around the null problem?